### DISCUSSION TITLE
API Basics tutorial or documentation

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#62] [Q&A]
@nquarder It would be great if you could share any resources about APIs and their various functions, like an introduction to getting started with API access. That would be awesome.

**Comment by nquarder** [comment]
Hi @DharaniSureshBabu - Yes I plan to make a recording that goes through setting up the API call from NOAA NCEI. Sorry I am a bit behind!

For now here is a link to our classic learning portal with a more general lesson on API to get you started: https://www.earthdatascience.org/courses/use-data-open-source-python/intro-to-apis/apis-in-python/
